app/init.php

```diff
@@ -196,6 +196,7 @@
 const METRIC_TEAMS = 'teams';
 const METRIC_USERS = 'users';
 const METRIC_MESSAGES  = 'messages';
+const METRIC_MESSAGES_COUNTRY_CODE  = '{countryCode}.messages';
```

Comment:
```suggestion
const METRIC_MESSAGES_COUNTRY  = '{countryCode}.messages';
```

Code can be the value, we don't need as a prefix.